tests/unit/Issue/IssueTest.php

```diff
@@ -50,34 +50,6 @@ public function tearDown()
         Mockery::close();
     }
 
-    /**
-     * @expectedException PHPUnit_Framework_Error
-     * @expectedExceptionMessage must implement interface StaticReview\Review\ReviewInterface
-     */
-    public function testConstructWithInvalidReview()
```

Comment:
Because the exception being throw in 7 is a TypeError, so we'd have to do a switch on php version before setting the expected exception.

Didn't seem worth it for a low risk test on type hinting.